.github/workflows/CI.yml

```diff
@@ -103,3 +103,10 @@ jobs:
         run: |
           VALGRIND_CMD="pyvrp pyvrp/tests/data/ORTEC-VRPTW-ASYM-0bdff870-d1-n458-k35.txt --max_runtime 0.5 --seed 1"
           poetry run env PYTHONMALLOC=malloc valgrind --tool=memcheck --quiet $VALGRIND_CMD
+      - if: matrix.compiler == 'clang' && matrix.python-version == '3.11'
+        # Run this only for one job to reduce workflow runtimes.
+        name: Run example notebooks
+        run: |
+          poetry install --with examples
+          poetry run jupyter nbconvert --to notebook --execute examples/cvrp.ipynb
+          poetry run jupyter nbconvert --to notebook --execute examples/vrptw.ipynb
```

Comment:
This is not covered by the cache. Can we update
```
      - name: Install Python dependencies
        if: steps.cache-python.outputs.cache-hit != 'true'
        run: poetry install --no-root
```
to include ```--with examples```? I'll clear the cached dependencies to make this work, let me know when.

Then,
```
          poetry run jupyter nbconvert --to notebook --execute examples/cvrp.ipynb
          poetry run jupyter nbconvert --to notebook --execute examples/vrptw.ipynb
```
This needs to be updated every time we have a new notebook. Maybe
```
for f in examples/*.ipynb; do poetry run jupyter nbconvert --to notebook --execute $f; done